go/vt/sqlparser/ast.go

```diff
@@ -1276,51 +1264,38 @@ func (node *ColName) Equal(c *ColName) bool {
 	return node.Name.Equal(c.Name) && node.Qualifier.Equal(c.Qualifier)
 }
 
+// ValTuple represents a list of value expressions.
+// It can be RowTuple or ParenExpr.
+type ValTuple interface {
+	iRowTuple()
+	Expr
+}
+
+func (RowTuple) iRowTuple()  {}
+func (ParenExpr) iRowTuple() {}
```

Comment:
I think it should work if you changed the return type of row_tuple to expr. I'll try to play around with it if that doesn't work.